ThemeDecorator/ThemeDecorator.js

```diff
@@ -203,7 +212,7 @@ const CustomizableSkinStyle = kind({
  */
 const ThemeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	// TODO: Document props passable to hoc ()
-	const {customSkin, float, i18n, noAutoFocus, overlay, ri, skin, spotlight} = config;
+	const {customSkin, float, i18n, noAutoFocus, overlay, ri, skin, spotlight, disableFullscreen} = config;
 
 	const bgClassName = classnames(
 		'enact-fit',
```

Comment:
Note: we may need to revisit to check if this line also needs to be conditionally applied or not. Related: PLAT-121194.